docs/developer/testing-environment.md

```diff
@@ -0,0 +1,52 @@
+# Setup Kubeapps testing environment
+
+This guide explains how to setup your environment to test Kubeapps integration with other services.
+
+## Background
+
+Kubeapps can be integrated with other services to extend its capabilities. Find more information about these integrations in the links below:
+
+- [Using Private App Repositories with Kubeapps](../user/private-app-repository.md).
+- [Kubernetes Service Catalog Kubeapps Integration](../user/service-catalog.md).
+
+This guide aims to provide the instructions to easily setup the environment to test these integrations.
+
+## Prerequisites
+
+- [Kubernetes cluster (v1.12+)](https://kubernetes.io/docs/setup/pick-right-solution/).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- [Helm client](https://helm.sh/docs/intro/install/).
+
+## Environment setup
+
+We are providing scripts to automatically setup both Kubeapps and the services to integrate on a K8s cluster. Find them under the [script](../../script) directory.
+
+Currently supported integrations:
+
+- Kubeapps integration with Harbor.
+
+### Kubeapps integration with Harbor
+
+You can setup environment to test Kubeapps integration with Harbor using the scripts below:
+
+- [setup-kubeapps](../../script/setup-kubeapps.sh).
+- [setup-harbor](../../script/setup-harbor.sh).
+
+These scripts will create the necessary namespaces, install the charts, wait for them to be available, and perform any extra action that might be needed. Find detailed information about how to use these scripts running the commands below:
+
+```bash
+./setup-kubeapps.sh --help
+./setup-harbor.sh --help
+```
+
+You can also use the [setup-kubeapps-harbor](../../script/setup-kubeapps-harbor.sh) script which is a wrapper that uses both the scripts mentioned above with some default values:
```

Comment:
I'd reverse the order here, and mention setup-kubeapps-harbor.sh first, as the wrapper, then the more detailed ones if you want them. But absolutely fine either way.